packages/react/src/date-range-picker/DateRangePicker.tsx

```diff
@@ -160,8 +164,12 @@ export const DateRangePicker = ({
 
 	const valueAsDateOrUndefined = useMemo(
 		() => ({
-			from: typeof value.from === 'string' ? undefined : value.from,
-			to: typeof value.to === 'string' ? undefined : value.to,
+			from:
+				typeof value.from === 'string'
```

Comment:
So you still need to make this change to date-picker.tsx L202?

```
const valueAsDateOrUndefined = typeof value === 'string' ? normaliseDateString(value) : value;
```

And then move `normaliseDateString` into date-picker/utils.ts?